source/ocsp-support/ocsp-support.rst

```diff
@@ -3,14 +3,14 @@ OCSP Support
 ============
 
 :Spec Title: OCSP Support
-:Spec Version: 1.0.2
+:Spec Version: 1.2.0
```

Comment:
Either 1.0.3 or 1.1.0, not 1.2.0.

Reply:
Ah sorry for the confusion: the current version of the spec is at 1.1.0, so this PR should bump it to 1.2. (I haven't rebased since I don't want to screw up force push during a review) https://github.com/vincentkam/specifications/blob/ocsp-support/source/ocsp-support/ocsp-support.rst